output/index.yaml

```diff
@@ -0,0 +1,110 @@
+openapi: 3.0.0
+info:
+  title: Sentences API
+  description: Sentences REST API
+  version: 0.1.1
+servers:
+- url: https://virtserver.swaggerhub.com/MHAMLIN/sentences/0.1.1
+  description: SwaggerHub API Auto Mocking
+paths:
+  /sentences:
+    get:
+      description: Returns a list of sentences
+      responses:
+        "200":
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Sentence'
+components:
+  schemas:
+    Sentence:
+      required:
+      - id
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the sentence
+          format: uuid
+        state:
+          pattern: "^\\d{2}"
+          type: string
+          description: State FIPS code
+        county:
+          pattern: "^\\d{5}"
+          type: string
+          description: County FIPS code
+        date:
+          type: string
+          description: "Sentencing date (month, day, and year)"
+          format: date
+        fine:
+          $ref: '#/components/schemas/SentenceTypeWithFines'
+        restitution:
+          $ref: '#/components/schemas/SentenceTypeWithFines'
+        confinement:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        suspension:
+          $ref: '#/components/schemas/SentenceType'
+        probation:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        community_service:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        treatment:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        death:
+          $ref: '#/components/schemas/SentenceType'
+        data_source:
+          $ref: '#/components/schemas/Sentence_data_source'
+    SentenceType:
+      type: object
+      properties:
+        hasType:
+          type: boolean
+    SentenceTypeWithDuration:
+      allOf:
+      - $ref: '#/components/schemas/SentenceType'
+      - type: object
+        properties:
+          minimum_duration:
+            type: integer
+          maximum_duration:
+            type: integer
+          duration_type:
```

Comment:
I am not opposed to this, but the last conversation we had I believe SEARCH was wanting us to just standardize and report everything back in our response in days. I am fine to keep it and add support (or not) if we find a need for it.

Reply:
The duration_type could also be defined as optional and we could just say that if it's missing, the specified durations could be assumed to be in days.